### PR TITLE
Replace deprecated update guidance

### PR DESCRIPTION
**Pull request by flinchbot**
We now use the MTR-Update script to update Teams Rooms. Also removed reference to incorrect information that patches are installed at 2:00AM.

@cazawideh @Travis-Snoozy

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-operations.md [hunk 1]
```diff
@@ -184,18 +184,9 @@ Copy-Item $movefile $targetDevice
...
-2. Run the following script targeting the Microsoft Teams Rooms devices, changing \<share\> to the device share as appropriate:
-    
-    ```PowerShell
-    Add-AppxPackage -Update -ForceApplicationShutdown -Path '\\<share>\$oem$\$1\Rigel\x64\Ship\AppPackages\*\*.appx' -DependencyPath (Get-ChildItem '\\<share>\$oem$\$1\Rigel\x64\Ship\AppPackages\*\Dependencies\x64\*.appx' | Foreach-Object {$_.FullName})
-    ```
+Microsoft Teams Rooms connects to Windows Update to retrieve operating system and peripheral device firmware updates.
+ 
+If you must manage updates manually and are also unable to follow the normal procedure for [Microsoft Store for Business](https://businessstore.microsoft.com/store) to [Distribute offline apps](/microsoft-store/distribute-offline-apps), you can acquire MTR-Update packages to perform app updates on supported OSes. The MTR-Update release may lag behind the store release, and might not always match the latest available build. Refer to the article [Manually Update a Microsoft Teams Rooms device](manual-update.md) for further details.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-operations.md:189]:
```suggestion
If you must manage updates manually and are also unable to follow the normal procedure for [Microsoft Store for Business](https://businessstore.microsoft.com/store) to [Distribute offline apps](/microsoft-store/distribute-offline-apps), you can acquire MTR-Update packages to perform app updates on supported operating systems. The MTR-Update release may lag behind the store release, and might not always match the latest available build. Refer to the article [Manually Update a Microsoft Teams Rooms device](manual-update.md) for further details.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @flinchbot, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cazawideh** [comment]
Re-doing this PR in private repo

**Pull request closed without merging**